password_policies/forms/validators.py

```diff
@@ -145,7 +145,7 @@ class BidirectionalValidator(BaseRFC4013Validator):
     #: The validator's error code.
     code = u"invalid_bidirectional"
     #: The validator's error message.
-    message = _("The new password contains ambiguious bidirectional characters.")
+    message = _("The new password contains ambiguous bidirectional characters.")
```

Comment:
This needs to be updated in other translations as well... Find & replace would be enough I guess.